### PR TITLE
fix: Not passing additional data in request

### PR DESCRIPTION
**Pull request by victorrabelozup**
### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/58fe7c188e149ee1fab14c18fbac1b12cca283b8/iOS/Sources/Beagle/Sources/Networking/Repository.swift#L172">iOS/Sources/Beagle/Sources/Networking/Repository.swift#L172</a> - TODOs should be resolved (change loadFromTextError insid…).<br />
<code>todo</code> <code>Repository.swift:172</code></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_ios">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: iOS/Sources/Beagle/Sources/Networking/Tests/RepositoryTests.swift [hunk 1]
```diff
@@ -150,6 +150,38 @@ final class RepositoryTests: XCTestCase {
             return
         }
     }
+    
+    func test_whenRequestSucceeds_withValidData_itShouldReturnSomeComponent2() {
```
Comment (lucasfonsecazup) [inline iOS/Sources/Beagle/Sources/Networking/Tests/RepositoryTests.swift:154]: We currently agree on iOS that we will use the following test title pattern: https://github.com/ZupIT/beagle/wiki/Swift-Style-Guide#tests

**Review by lucasfonsecazup — COMMENTED** [review]
Nice

File: iOS/Sources/Beagle/Sources/Networking/Tests/RepositoryTests.swift (hunk 1 above)
Comment (lucasfonsecazup) [reply iOS/Sources/Beagle/Sources/Networking/Tests/RepositoryTests.swift]: Suggestion : testRequestSuccessValidData

**Review by theffc — APPROVED** [review]

**Review by dantes-git — APPROVED** [review]

**Review by caioortu — APPROVED** [review]

**Pull request merged**